contrib/ddsketch_calc.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+#
+# ddsketch_calc.py
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2013-2019 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import math as m
+    
+class DDSketch(object):
+    A = 6.0 / 35.0
+    B = -3.0 / 5.0
+    C = 10.0 / 7.0
+    EPS = 1e-18
+    correctingFactor = 1.00988652862227438516
+    offset = 0
+    multiplier = 0
+    gamma = 0
+
+    def __init__(self, errorGuarantee):
+        self.gamma = (1 + errorGuarantee) / (1 - errorGuarantee)
```

Comment:
nit: add references to origin of the equations and math referred here as well in the code below? To me it is not very understandable the motivation just reading the python math equations